pandas/io/formats/style.py

```diff
@@ -1870,7 +1870,7 @@ def apply_index(
 
         >>> df = pd.DataFrame([[1,2], [3,4]], index=["A", "B"])
         >>> def color_b(s):
-        ...     return {ret}
+        ...     return '{ret}'
```

Comment:
I think this one is a false-positive from flake8, it shows up fine in the docs

https://pandas.pydata.org/docs/dev/reference/api/pandas.io.formats.style.Styler.apply_index.html?highlight=apply_index

Reply:
You're right, I will ignore the undefined error for this line